README.md

```diff
@@ -15,6 +15,14 @@ Other resources: [consul.io/community](https://www.consul.io/community)
 
 Additionally, for issues and pull requests, we'll be using the 👍 reactions as a rough voting system to help gauge community priorities. So please add 👍 to any issue or pull request you'd like to see worked on. Thanks.
 
+## Roadmap
+
+Knowing about our upcoming features and priorities helps our users plan. This repository contains information about what we are working on and allows all our user to give direct feedback. Please visit the [Roadmap FAQs](#roadmap-faqs) for more information on the categorization of the roadmap,
```

Comment:
```suggestion
Knowing about our upcoming features and priorities helps our users plan. This repository contains information about what we are working on and allows all our user to give direct feedback. Please visit the [Roadmap FAQs](#roadmap-faqs) for more information on the categorization of the roadmap.
```

Reply:
not sure if the comma was intentional or if the period is better suited. feel free to address at your convenience